tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -404,55 +374,23 @@ Change methodology status to Approved
 
 Create amendment
     [Tags]  HappyPath
-    user clicks link  Home
-    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
-    user waits until page contains accordion section   ${PUBLICATION_NAME}
-    user opens accordion section  ${PUBLICATION_NAME}
-    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
-    user opens details dropdown   ${RELEASE_NAME} (Live - Latest release)  ${accordion}
-    ${details}=  user gets details content element  ${RELEASE_NAME} (Live - Latest release)  ${accordion}
-    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
-    user clicks button  Amend this release  ${details}
-    user clicks button  Confirm
-
+    user creates amendment for release  ${PUBLICATION_NAME}  ${RELEASE_NAME}  (Live - Latest release)
+    
 Replace data files for amendment
     [Tags]  HappyPath
-    [Documentation]  EES-1442
-    user clicks link  Data and files
-    user waits until page contains element  id:dataFileUploadForm-subjectTitle
-    user waits until h2 is visible  Uploaded data files
-    user waits until page contains accordion section   ${SUBJECT_NAME}
-    user opens accordion section   ${SUBJECT_NAME}
-    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
-    user clicks link  Replace data  ${section}
-    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}dates.csv
-    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}dates.meta.csv
-    user clicks button  Upload data files
-    user waits until page contains  Footnotes: ERROR  120
-    user opens details dropdown  Footnote 2 ${SUBJECT_NAME}
-    user clicks button  Delete footnote
-    user clicks button  Confirm
-    #EES-1442: Bug when Confirm data replacement button doesn't show
-    user reloads page
-    user waits until page contains  Footnotes: OK
-    user waits until page contains  Data blocks: OK
-    user waits until button is enabled  Confirm data replacement
+    user replaces data files for amendment   ${SUBJECT_NAME}  dates.csv  dates.meta.csv
+    
 
 Confirm data replacement
     [Tags]  HappyPath
     user clicks button  Confirm data replacement
 
 Delete second subject file
     [Tags]  HappyPath
+    # to avoid focus issues 
```

Comment:
I'd put this comment on the same line it refers to: `user clicks link Footnotes  # to avoid focus issues`

This just means that if someone alters this test in the future, hopefully the comment won't get left behind and end up no longer making any sense.